### ISSUE TITLE
bug: iam:GetUser returns an error 500

### ISSUE THREAD

**Issue by christophetd** [localstack/localstack#5196]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Running `aws --endpoint-url=http://localhost:4566 iam get-user` returns:

```
Unable to parse response (syntax error: line 1, column 54), invalid XML received. Further retries may succeed:
b'<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n<title>500 Internal Server Error</title>\n<h1>Internal Server Error</h1>\n<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>\n'
```

### Expected Behavior

Return the actual data and not error 500.

### How are you starting LocalStack?

With the `localstack` script

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

```
localstack start
```

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

```
aws --endpoint-url=http://localhost:4566 iam get-user
```

### Environment

```markdown
AWS CLI version: `aws-cli/2.4.6`
Localstack version: `0.13.2`
```

### Anything else?

Debug output:

```
2021-12-28T09:52:06.017:DEBUG:plugin.manager: instantiating plugin PluginSpec(localstack.aws.provider.iam:default = <function iam at 0x7fb864599310>)
2021-12-28T09:52:06.017:DEBUG:plugin.manager: loading plugin localstack.aws.provider:iam:default
2021-12-28T09:52:06.230:INFO:localstack.services.infra: Starting mock IAM service on http port 4566 ...
2021-12-28T09:52:06.231:INFO:localstack.services.motoserver: starting moto server on http://0.0.0.0:41753
2021-12-28T09:52:06.232:DEBUG:localstack.services.plugins: checking service health iam:41753
2021-12-28 09:52:06,267:API: Error on request:
Traceback (most recent call last):
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/werkzeug/serving.py", line 319, in run_wsgi
execute(self.server.app)
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/werkzeug/serving.py", line 308, in execute
application_iter = app(environ, start_response)
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/moto/server.py", line 239, in __call__
return backend_app(environ, start_response)
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/app.py", line 2091, in __call__
return self.wsgi_app(environ, start_response)
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/app.py", line 2076, in wsgi_app
response = self.handle_exception(e)
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function
return cors_after_request(app.make_response(f(*args, **kwargs)))
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/app.py", line 2073, in wsgi_app
response = self.full_dispatch_request()
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/app.py", line 1518, in full_dispatch_request
rv = self.handle_user_exception(e)
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function
return cors_after_request(app.make_response(f(*args, **kwargs)))
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/app.py", line 1516, in full_dispatch_request
rv = self.dispatch_request()
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/flask/app.py", line 1502, in dispatch_request
return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
File "/opt/code/localstack/localstack/utils/aws/request_context.py", line 161, in convert_flask_to_httpretty_response_call
return convert_flask_to_httpretty_response_call_orig(*args, **kwargs)
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/moto/core/utils.py", line 147, in __call__
result = self.callback(request, request.url, dict(request.headers))
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/moto/core/responses.py", line 203, in dispatch
return cls()._dispatch(*args, **kwargs)
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/moto/core/responses.py", line 313, in _dispatch
return self.call_action()
File "/opt/code/localstack/.venv/lib/python3.8/site-packages/moto/core/responses.py", line 403, in call_action
response = method()
File "/opt/code/localstack/localstack/services/iam/iam_starter.py", line 147, in iam_response_get_user
user = moto_iam_backend.users[user_name]
KeyError: 'default_user'
``` 

Appears to be coming from https://github.com/localstack/localstack/blob/master/localstack/services/iam/iam_starter.py#L147

**Comment by whummer** [comment]
Thanks for reporting @christophetd . This should be fixed in #5206 , can you please pull the `latest` Docker image and give it another try? 

We have also added new config options `TEST_IAM_USER_ID` and `TEST_IAM_USER_NAME` which you can use to customize the default user returned by IAM GetUser or STS GetCallerIdentity (we'll update our docs soon with these new configs). Hope that helps - please keep us posted how it goes.. Thanks!

**Comment by christophetd** [comment]
Looks good! Thank you for the quick fix!
